src/components/containers/AdminDashboard/AdminWorkshopsContainer.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import WorkshopsOverview from "../../presentational/AdminDashboard/WorkshopsOverview";
+import CenteredTabs from "../../presentational/CenteredTabs";
+import { Typography } from "@material-ui/core";
+import { RouteComponentProps } from "react-router";
+
+type AdminWorkshopsContainerProps = RouteComponentProps<any> & {};
+
+const workshopTabs = [
+	{
+		title: "Current"
```

Comment:
Does this need to be an array of objects?  If we're only using 1 value wouldn't it be simpler to just have an array of strings?